src/containers/AccessPointDetails/components/Status/index.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { Card, Form, Table } from 'antd';
+import PropTypes from 'prop-types';
+
+import styles from '../../index.module.scss';
+
+const Status = ({ data }) => {
+  const { Item } = Form;
+
+  const layout = {
+    labelCol: { span: 5 },
+    wrapperCol: { span: 12 },
+  };
+
+  const columns = [
+    {
+      title: 'Severity	',
+      dataIndex: 'severity',
+      key: 'severity',
+    },
+    {
+      title: 'Type',
+      dataIndex: 'alarmCode',
+      key: 'type',
+    },
+    {
+      title: 'Message	',
+      dataIndex: ['details', 'message'],
+      key: 'security',
+    },
+    {
+      title: 'Timestamp',
+      dataIndex: 'createdTimestamp',
+      key: 'timestamp',
+      render: time => {
+        const date = new Date(parseInt(time, 10));
+        return date.toLocaleString();
+      },
+    },
+  ];
+
+  return (
+    <>
+      <Form {...layout}>
+        <Card title="Status">
+          <Item label=" " colon={false}>
+            <div className={styles.InlineDiv}>
+              <p>2.4GHz</p>
+              <p>5 GHzU</p>
+              <p>5 GHzL</p>
```

Comment:
AP's can have a different combination of radios, is2dot4GHz, is5GHz, is5GHzU, or is5GHzL, all radio based display and forms need to be dynamic based on the specific AP radioMaps